chapter_attention-mechanisms/transformer.md

```diff
@@ -787,6 +1028,12 @@ d2l.show_heatmaps(
     figsize=(7, 3.5))
 ```
 
+```{.python .input}
+#@tab tensorflow
+d2l.show_heatmaps(enc_attention_weights, xlabel='Key positions', ylabel='Query positions',
```

Comment:
can be merged with mxnet code

Reply:
Will fix this and other similar issues soon.